templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -112,14 +112,14 @@ objects:
   metadata:
     name: access-asb-role
   rules:
-  - nonResourceURLs: ["${BROKER_URL_PREFIX}", "${BROKER_URL_PREFIX}/*"]
+  - nonResourceURLs: ["/${BROKER_URL_PREFIX}", "/${BROKER_URL_PREFIX}/*"]
```

Comment:
Pretty sure if you do this you need to update: https://github.com/openshift/ansible-service-broker/pull/799/files#diff-54e88cebcb04cc49b8796b2f774cb6c7R425

Reply:
In my testing this works.

However, I've also changed the way I pass in the parameter here:
https://github.com/openshift/ansible-service-broker/pull/799/files#diff-45df7e79b5e14d6545c816ae8b2dc180R61

and also made changes here:
https://github.com/openshift/ansible-service-broker/pull/799/files#diff-54e88cebcb04cc49b8796b2f774cb6c7L400

Reply:
it's worth noting that with the above changes, I've also had to always add this line to the AWS template counter part here:
https://github.com/openshift/ansible-service-broker/pull/799/files#diff-54e88cebcb04cc49b8796b2f774cb6c7R348

Reply:
Because of your testing with the change you are setting the correct value.  If you were to run this with default parameter for BROKER_URL then it would cause issue.

Reply:
ahhh aplogies.. my fault.  I will update that